bar-trender-front-end/src/components/FreeChart.js

```diff
@@ -5,13 +5,44 @@ class BeerChart extends Component{
   constructor(props){
     super(props);
     this.state =this.props.data;
+   
   }
 
+
+  
+
   render(){
+
+       var graph2 = {
+      chartData:{
+        labels: ['Billar', 'Pista de Baile', 'Futbolin', 'Otros'],
+        datasets:[
+          {
+            label:'Busquedas',
+            data:[
+              111455,
+              23444,
+              13060,
+              331,
+            ],
+            backgroundColor:[
+              'rgba(255, 99, 132, 0.6)',
+              'rgba(54, 162, 235, 0.6)',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7173304)
```suggestion
              "rgba(54, 162, 235, 0.6)",
```